include/onnxruntime/core/framework/allocator.h

```diff
@@ -39,6 +39,9 @@ constexpr const char* CUDA = "Cuda";
 constexpr const char* CUDA_PINNED = "CudaPinned";
 constexpr const char* MIGRAPHX = "MIGraphX";
 constexpr const char* MIGRAPHX_PINNED = "MIGraphXPinned";
+constexpr const char* OpenVINO = "OpenVINO";
+constexpr const char* IGPU = "IGPU";
```

Comment:
would it be better to have something like OpenVINO_GPU instead of IGPU?
DirectML can also access integrated gpu so that might cause confusion.

Reply:
I have changed, Please verify